core/tests/data/dummy_ci_tests/.dummy_travis.yml

```diff
@@ -0,0 +1,14 @@
+env:
+  jobs:
+    - RUN_E2E_TESTS_ACCESSIBILITY=true
```

Comment:
Let's keep this and other test files as minimal as possible. Have just two jobs: RUN_E2E_TESTS_ONEWORD and RUN_E2E_TESTS_TWO_WORDS and leave it at that.

Have just as many examples needed to test stuff, no more than that.

Reply:
Done.